pgrx/src/spi/cursor.rs

```diff
@@ -67,18 +67,14 @@ pub struct SpiCursor<'client> {
     pub(crate) client: &'client SpiClient,
 }
 
-impl SpiCursor<'_> {
+impl<'client> SpiCursor<'client> {
     /// Fetch up to `count` rows from the cursor, moving forward
     ///
     /// If `fetch` runs off the end of the available rows, an empty [`SpiTupleTable`] is returned.
-    pub fn fetch(&mut self, count: libc::c_long) -> std::result::Result<SpiTupleTable, SpiError> {
-        // SAFETY: no concurrent access
-        unsafe {
-            pg_sys::SPI_tuptable = std::ptr::null_mut();
-        }
```

Comment:
There were a couple of these (when I suppose there "should" have been three!).  It's not necessary, however.  Postgres does this for us in its SPI_cursor_fetch and SPI_execute operation functions.